containers/jetty/src/main/java/com/yahoo/athenz/container/AthenzJettyContainer.java

```diff
@@ -368,7 +368,7 @@ void loadServicePrivateKey() {
         final String excludedProtocols = System.getProperty(AthenzConsts.ATHENZ_PROP_EXCLUDED_PROTOCOLS, ATHENZ_DEFAULT_EXCLUDED_PROTOCOLS);
         final boolean renegotiationAllowed = Boolean.parseBoolean(System.getProperty(AthenzConsts.ATHENZ_PROP_RENEGOTIATION_ALLOWED, "false"));
 
-        SslContextFactory.Server sslContextFactory = new SslContextFactory.Server();
```

Comment:
Override SslContextFactory.Server with our own version that print certificates on failures